src/machine/machine_atmega.go

```diff
@@ -12,12 +12,16 @@ func (p Pin) Configure(config PinConfig) {
 	if config.Mode == PinOutput { // set output bit
 		if p < 8 {
 			avr.DDRD.SetBits(1 << uint8(p))
+		} else if p > 13 {
+			avr.DDRC.SetBits(1 << uint8(p-14))
```

Comment:
Can you maybe process the pin in the pin order, for readability? Something like this:

```go
if p < 8 {
    // ...
} else if p < 14 {
    // ...
} else {
    // ...
}
```

The same goes for the other 3 changes.

Reply:
Thank you for your review. I fixed. Please review again.